src/urllib3/util/ssl_.py

```diff
@@ -317,6 +321,11 @@ def create_urllib3_context(
         # there is a risk associated with it being on wire,
         # if the server is not rotating its ticketing keys properly.
         options |= OP_NO_TICKET
+        # In python 3.13+ ssl.create_default_context() sets VERIFY_X509_PARTIAL_CHAIN
+        # and VERIFY_X509_STRICT so we do the same
+        if sys.version_info >= (3, 13):
+            options |= VERIFY_X509_PARTIAL_CHAIN
+            options |= VERIFY_X509_STRICT
```

Comment:
I believe the flags should be set as [`SSLContext.verify_flags`](https://docs.python.org/3/library/ssl.html#ssl.SSLContext.verify_flags), not as options.
I suggest adding a `verify_flags` parameter to `create_urllib3_context` too.

Reply:
Thanks, I knew something was "off" with that first attempt. :-)